src/components/external-link/external-link.css

```diff
@@ -0,0 +1,4 @@
+.external-link-icon {
+  position: relative;
+  top: 1.5px;
+}
```

Comment:
You're right, we still have a need for a little CSS to make it look right.

Try this instead:
```
.external-link-icon {
  vertical-align: middle;
}
```